more_itertools/more.py

```diff
@@ -805,6 +806,13 @@ def interleave_longest(*iterables):
     return filter(lambda x: x is not _marker, i)
 
 
+# The performance of roundrobin is far superior to interleave_longest in PyPy
+if python_implementation() == 'PyPy':
+    _interleave_longest_docstring = interleave_longest.__doc__
+    interleave_longest = roundrobin
+    interleave_longest.__doc__ = _interleave_longest_docstring
```

Comment:
Wouldn't that change `roundrobin`s `__doc__` because `interleave_longest = roundrobin` just creates an alias.